setup.py

```diff
@@ -21,7 +21,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 NAME = "platform-api-python-client"
-VERSION = "0.3.1"
```

Comment:
There was no major release so it should be still 0.3.x. I think the next version is 0.3.2

Reply:
ok